components/04-views/product-view/_product-view.scss

```diff
@@ -307,7 +313,10 @@ $product-view__wrapper-margin-bottom\@medium            : 40px !default;
     &__rating {
         color: $product-view__rating-text-color;
         clear: both;
```

Comment:
Additionally I think we don't need ` clear: both;`

Reply:
done